lib/cli/options.js

```diff
@@ -69,7 +70,7 @@ module.exports = yargs
       robot.printError(error || message);
     } else {
       yargs.showHelp();
-      errors.print(error || message);
+      utils.printError(error || message);
```

Comment:
Makes a lot of sense as well. I updated the `utils` and the `robot` sides of this :)